org.eclipse.m2e.pde.target/src/org/eclipse/m2e/pde/target/MavenTargetBundle.java

```diff
@@ -26,12 +26,15 @@
 import org.eclipse.core.runtime.Status;
 import org.eclipse.equinox.frameworkadmin.BundleInfo;
 import org.eclipse.pde.core.target.TargetBundle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import aQute.bnd.osgi.Analyzer;
 import aQute.bnd.osgi.Jar;
 
 public class MavenTargetBundle extends TargetBundle {
 
+	private static final Logger LOGGER = LoggerFactory.getLogger(MavenTargetBundle.class);
```

Comment:
Would you mind using the Eclipselog (`ILog`) instead here, that would make it more accessible I think and users are more aware of that facility.